scripts/scil_volume_pairwise_comparison.py

```diff
@@ -0,0 +1,206 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""
+Evaluate pair-wise similarity measures of masks and atlas.
+All volumes must be in the same space.
+
+For the voxel representation, the computed similarity measures are:
+    adjacency_voxels, dice_voxels, volume_overlap, volume_overreach.
+
+This can work for BET mask, WMPARC, bundle label maps. The datatype of the
+input volumes must be uint8 (mask) or uint16 (label map and atlas).
+"""
+
+
+import argparse
+import itertools
+import json
+import logging
+import multiprocessing
+
+from dipy.io.utils import get_reference_info
+import numpy as np
+
+from scilpy.io.image import load_img
+from scilpy.io.utils import (add_json_args,
+                             add_overwrite_arg,
+                             add_processes_arg,
+                             add_reference_arg,
+                             add_verbose_arg,
+                             assert_headers_compatible,
+                             assert_inputs_exist,
+                             assert_outputs_exist,
+                             validate_nbr_processes)
+from scilpy.image.labels import get_data_as_labels
+from scilpy.tractanalysis.reproducibility_measures import compute_dice_voxel, \
+    compute_bundle_adjacency_voxel
+
+
+def _build_arg_parser():
+    p = argparse.ArgumentParser(
+        description=__doc__,
+        formatter_class=argparse.RawTextHelpFormatter)
+    p.add_argument('in_volumes', nargs='+',
+                   help='Path of the input volumes.')
+    p.add_argument('out_json',
+                   help='Path of the output json file.')
+
+    p.add_argument('--adjency_no_overlap', action='store_true',
+                   help='If set, do not count zeros in the average BA.')
```

Comment:
What is BA? Not defined anywhere.

I think the variable's name is not super clear. `--ignore_zeros_in_BA, help="If set, do not count zeros in the average bundle adgency (BA)`?

Reply:
Actually, the word "adjacency" is only used in these two scripts (based on scil_search_keywords). I'm not even sure what it means. Could you define it?